pages/linux/pw-top.md

```diff
@@ -0,0 +1,29 @@
+# pw-top
+
+> View the pipewire nodes and devices statistics in real-time.
+> See also: `pipewire`, `pw-dump`, `pw-cli`, `pw-profiler`.
+> More information: <https://docs.pipewire.org/page_man_pw-top_1.html>.
+
+- Display an interactive view of pipewire nodes and devices:
+
+`pw-top`
+
+- Monitor a remote instance:
+
+`pw-top --remote {{remote_name}}`
+
+- Print information multiple times instead of running in interactive mode:
+
+`pw-top --batch-mode`
+
+- Print information a specific number of times:
+
+`pw-top --batch-mode --iterations {{3}}`
+
+- Display help:
+
+`pw-top --help`
+
+- Display version:
+
+`pw-top --version`
```

Comment:
Hmm. What's the policy on informational/conventional flags like these? I feel like they aren't particularly useful - if someone is using TLDR, I think it's fair to assume they likely know the existence of these conventional flags. But I do see them quite a bit in existing pages (`rg -- '--help'`).

Reply:
Agreed, we can remove them here. The help and version commands are particularly useful in Windows pages as there are a lot of differences/implementations for them depending upon the shell/utility.

Reply:
Yep, I think we should keep TLDR pages as minimal and useful as possible.
There's nothing wrong on removing these options (`--help` and `--version`) @vitorhcl.

Reply:
They are filler in some regard however there merit for newer users to Linux especially can be helpful. They are the first commands to be replace if better commands are needed, I think they should remain.